hack/test/e2e-firecracker.sh

```diff
@@ -7,9 +7,10 @@ source ./hack/test/e2e.sh
 PROVISIONER=firecracker
 CLUSTER_NAME=e2e-${PROVISIONER}
 
-case "${REGISTRY:-false}" in
-  registry.ci.svc:5000)
-    REGISTRY_ADDR=`python -c "import socket; print socket.gethostbyname('registry.ci.svc')"`
+case "${CI:-false}" in
```

Comment:
Simplify things a bit with less environment variables. Just use the `CI` env var instead since we know that we have a regsitry on `127.0.0.1`.